ops/cmd/import_devnet/main.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/state"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/config"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/generate"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/output"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/paths"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	StateFilename = &cli.StringFlag{
+		Name:      "state-filename",
+		Usage:     "Filename of an op-deployer state file.",
+		Required:  true,
+		TakesFile: true,
+	}
```

Comment:
Might want to change this to `StatePath` for consistency with the `ManifestPath` flag

Reply:
[6e560c2](https://github.com/ethereum-optimism/superchain-registry/pull/1011/commits/6e560c2b2846baa452917d379392e88d51dcae09)